examples/lookup/lookup.cpp

```diff
@@ -181,6 +181,7 @@ int main(int argc, char ** argv){
                         const int startIdx = i + ngram_size;
                         const int endIdx = startIdx + n_draft;
                         if (endIdx < inp_size) {
+                            draft.reserve(endIdx - startIdx);
```

Comment:
I don't think draft is necessarily empty here:
```suggestion
                            draft.reserve(draft.size() + endIdx - startIdx);
```